### DISCUSSION TITLE
Using formal or informal language for Dutch translations?

### DISCUSSION THREAD

**Discussion by AnonymousWP** [rustdesk/rustdesk#5948] [Polls]
Related: https://github.com/rustdesk/rustdesk/pull/5947

In Dutch two options can be used:

1. "U"
2. "Je"

"U" is a formal way of saying "You", while "Je" is the informal way. Which one do you prefer? Right now, both are used interchangeably for the Dutch translations. For consistency purposes, which is preferred? I'll change it accordingly. @rustdesk I'd also like to get your opinion regarding this, if possible.

**Comment by rustdesk** [comment]
I prefer 1

**Comment by AnonymousWP** [comment]
Fixed in https://github.com/rustdesk/rustdesk/pull/5987.
